content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -1,17 +1,16 @@
 +++
-title = "Using Jupyter notebooks in kubeflow"
-description = "Why use Jupyter notebooks in Kubeflow"
+title = "Using Jupyter Notebooks in Kubeflow"
+description = "Why use Jupyter Notebooks in Kubeflow"
 weight = 30
 +++
 
-Why would you use Jupyter networks in Kubeflow?
 
-The benefit of integrating Jupyter notebooks service in Kubefflow is that:
+The benefit of integrating Jupyter Notebooks service in Kubeflow is that:
 
 * In an enterprise environment, it integrates well with the rest of the infrastructure with respect to authentication and access control, and
 * It allows users to create notebook containers / pods in the cluster rather than locally on their laptops.
 This makes it possible for admins to provide standard notebook images for their organization, and add Secrets/Credentials allowing the notebook users to access data from sources with appropriate RBAC.
 
-When you bundle Jupyter notebooks in Kubeflow, you can use the Fairing library, that helps to submit Training jobs using TFJob (that can run single node or distributed on the same K8s cluster, but not inside the notebook pod itself). This makes the docker containerization and the calls to talk to the k8s master for allocating pods for the job, transparent to data scientists.
+When you bundle Jupyter Notebooks in Kubeflow, you can use the Fairing library, that helps to submit Training jobs using TFJob (that can run single node or distributed on the same K8s cluster, but not inside the notebook pod itself). This makes the docker containerization and the calls to talk to the k8s master for allocating pods for the job, transparent to data scientists.
```

Comment:
- nit: You may want to break up the sentence, `When ..., that helps`.
- Does this mean that the Fairing library can be imported into the Jupyter Notebook, `you can use the Fairing library`?
- Spell out Kubernetes instead of k8s/K8s
- Capitalize Job; is it the TFJob or another training job?
- Line 16, capitalize `Notebook`?

Reply:
- "When you bundle Jupyter Notebooks in Kubeflow, you can use the Fairing library to submit training jobs using TFJob. The training job can run single node or distributed on the same Kubernetes cluster, but not inside the notebook pod itself. Submitting the job with the Fairing library makes processes like Docker containerization and pod allocation clear for data scientists."